stateMatrix.m

```diff
@@ -3,9 +3,11 @@
 global TaskParameters
 
 %% Define ports
-LeftPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/1000,10));
-CenterPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/100,10));
-RightPort = floor(mod(TaskParameters.GUI.Ports_LMRAir/10,10));
+LeftPort = floor(mod(TaskParameters.GUI.Ports_LMRAudLRAir/100000,10));         
```

Comment:
I can't see the change of parameter name [there](https://github.com/ioannapandi/Mouse2AFC/pull/2/files#diff-d130cd7064a46e44fa26d39708b58ad5R26), so you need to update the variable name there and the default value of the parameter.